cmd/destroy.go

```diff
@@ -112,21 +116,17 @@ func destroy(ctx *cli.Context) error {
 
 		fmt.Printf(" volume name: %s\n", format.Name)
 		fmt.Printf(" volume UUID: %s\n", format.UUID)
-		fmt.Printf("data storage: %s://%s\n", format.Storage, format.Bucket)
+		fmt.Printf("data storage: %s\n", blob)
 		fmt.Printf("  used bytes: %d\n", totalSpace-availSpace)
 		fmt.Printf(" used inodes: %d\n", iused)
 		warn("The target volume will be destoried permanently, including:")
-		warn("1. objects in the data storage")
-		warn("2. entries in the metadata engine")
+		warn("1. ALL objects in the data storage")
```

Comment:
it's better to show the address of data storage here again

Reply:
added meta as well